webapp/loggedin/modules/analysis/processingChain/components/processingChainSteps.js

```diff
@@ -23,20 +24,27 @@ const ProcessingChainSteps = props => {
   const editingStep = useSelector(ProcessingStepState.isEditingStep)
   const dispatch = useDispatch()
 
-  const processingSteps = ProcessingChain.getProcessingSteps(processingChain)
+  // Fetch steps on mount
   useEffect(() => {
     if (!ProcessingChain.isTemporary(processingChain) && !editingStep) {
       dispatch(fetchProcessingSteps(ProcessingChain.getUuid(processingChain)))
     }
   }, [])
 
+  const processingSteps = ProcessingChain.getProcessingSteps(processingChain)
+  const lastStepHasCategory = Boolean(ProcessingStep.getCategoryUuid(R.last(processingSteps)))
```

Comment:
Should we add a method to peocessingStep hasCategory?

Reply:
good